java/find-address/README.md

```diff
@@ -1,30 +1,31 @@
-# Find Address
-Geocode an address and show it on a map view.
+# Find address
 
-![Find Address App](find-address.png)
+Find the location for an address.
+
+![Image of find address](find-address.png)
+
+## Use case
+
+A user can input a raw address into your app's search bar and zoom to the address location.
 
 ## How to use the sample
-Type in an address in the search view at the top of the screen. Suggestions will appear as text is entered. Tap a suggestion or enter your own text to see the address marked with a pin. Tapping on the pin will show the address in a callout.
+
```

Comment:
I'm working on this sample just now: Android has implemented it slightly differently to other SDKs in that we have an autopopulate for the addresses. Therefore we can just revert back to the original description here.